js/m.viewer.js

```diff
@@ -1333,7 +1349,7 @@ function showLabel(jval,lval) {
      _p.replaceWith(_nn);
       _p=jQuery('#dlink');
      if (typeof lval != "undefined") {
-       _nn='<a id="dlink" href="'+encodeURI(lval['url'])+'">'+lval['label']+'</a>'
+       _nn='<a id="dlink" href="'+safeEncodeURI(lval['url'])+'">'+lval['label']+'</a>'
```

Comment:
I went ahead and changed this function too, although I wasn't able to directly test if it made any difference. `addVolume()` is the only other place `encodeURI()` is called, but it doesn't appear to be called in this project. A similar file, `m.testviewer.js` appears to be a copy of this file and makes use of `addVolume(),` but that's only if this project is run from `all.html`.